patterns-modules/design-patterns-creational-2/pom.xml

```diff
@@ -20,5 +20,9 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+    
+    <properties>
+        <mockito.version>5.2.0</mockito.version>
```

Comment:
5.11.0?

Reply:
The latest version for [mockito-inline](https://mvnrepository.com/artifact/org.mockito/mockito-inline/5.2.0) is 5.2.0, hence the version has been forced in some child modules.